packages/botbuilder-adapter-web/src/web_adapter.ts

```diff
@@ -316,4 +316,15 @@ export class WebAdapter extends BotAdapter {
     public isConnected(user: string): boolean {
         return typeof clients[user] !== 'undefined';
     }
+
+    /**
+     * Returns websocket connection of given user
+     * @param user
+     */
```

Comment:
Can you add a little snippet of example code here showing how it can be used?

Reply:
The code I gave as example above is too long for docblock, isn't it?
A simple example would probably use `send`, `close` or `terminate`.

Something like this maybe?
```js
if (message.action === 'disconnect') { bot.controller.adapter.getConnection(message.user).terminate(); }
```

Reply:
Sure! that's great.

Reply:
Done